src/app/portlets/dot-edit-page/content/services/html/dot-edit-content-toolbar-html.service.spec.ts

```diff
@@ -41,7 +41,7 @@ describe('DotEditContentToolbarHtmlService', () => {
                 dummyContainer = testDoc.createElement('div');
                 const htmlElement: HTMLHtmlElement = testDoc.getElementsByTagName('html')[0];
                 dummyContainer.innerHTML = `
-                    <div data-dot-object="container">
+                    <div data-dot-object="container" data-dot-can-add="CONTENT,WIDGET,FORM">
```

Comment:
We need more testing here, what happen if we just get 'WIDGET,FORM' or just one of them